static/templyte/src/app/create-template/create-template.component.html

```diff
@@ -22,6 +22,8 @@
     <div class="container">
         <input class="form-control" id="template-name-input" matInput [(ngModel)]="templateName"
                placeholder="What is the name of your template?" value="My Awesome Template!">
+        <input class="form-control" id="template-description-input" matInput [(ngModel)]="templateDescription"
+               placeholder="Describe your template" value="This template is awesome and cool">
```

Comment:
What is value used for? Same question for template name.

Reply:
Fixed this, got rid of value.